benches/benchmark.rs

```diff
@@ -152,7 +152,10 @@ fn bench_search_borrowed_crl_small(c: &mut Bencher) {
         .unwrap()
         .into();
 
-    c.iter(|| black_box(assert!(matches!(crl.find_serial(FAKE_SERIAL), Ok(None)))));
+    c.iter(|| {
+        assert!(matches!(crl.find_serial(FAKE_SERIAL), Ok(None)));
+        black_box(())
```

Comment:
Maybe put the `black_box` around the `crl.find_serial()` expression? (Or perhaps just remove it entirely.)

Reply:
Done.